test/e2e/gameserver_test.go

```diff
@@ -87,8 +87,9 @@ func TestSDKSetLabel(t *testing.T) {
 		return gs.ObjectMeta.Labels != nil, nil
 	})
 
-	assert.Nil(t, err)
-	assert.NotEmpty(t, gs.ObjectMeta.Labels["agones.dev/sdk-timestamp"])
+	if assert.Nil(t, err) {
```

Comment:
if we're here, can we flip this to `assert.NoError()` - which will give us the error results if it fails.